src/ngraph/runtime/cpu/cpu_visualize_tree.cpp

```diff
@@ -46,7 +80,9 @@ namespace ngraph
             const visualize_tree_ops_map_t& get_visualize_tree_ops_map()
             {
                 const static visualize_tree_ops_map_t vtom{
-                    {TI(runtime::cpu::op::ConvertLayout), visualize_convert_layout}};
+                    {TI(runtime::cpu::op::ConvertLayout), visualize_convert_layout},
+                    //{TI(ngraph::op::Reshape), visualize_reshape}
```

Comment:
I added this to help me understand ReShape : I wanted to see the in/out formats in the pdf graphs because the Reshape was not obvious. 
I can uncomment it if it would be useful to others to see the formats for ReShape.
I kept it commented so that one can uncomment when necessary

Reply:
seems like it could be useful.  I would leave it in and folks can comment out if it is annoying.